### ISSUE TITLE
Synthetic data

### ISSUE THREAD

**Issue by eribul** [ropensci/coder#126]
- [x] One thing that's missing from the documentation (for functions, the README, and the vignettes) is an example of what someone would do with the output of the package. The example in the "coder" vignette isn't healthcare-related, instead focused on a toy example involving patient cars. Perhaps replace it with a vignette that works through a "real" example, and shows at least one result from it that would be similar to what your next typical step would be. Besides being more interesting, it serves as documentation for the output: if you created a histogram of Charlson indices, it brings the user's attention to the importance of that column.
You could add something from the result of your ex_people and ex_icd10, but that join has only a single positive result (one patient with peripheral vascular disease). Of course real healthcare data is necessarily private, but you could instead consider taking a small sample of rows and columns from the SynPUF data, which is synthetic emergency room data including admissions and diagnoses. After using coder to determine what diagnoses occurred after an emergency room visit, the vignette could get one or two results from the data (e.g. showing the average Charlson comorbidity index of emergency room admissions over time, creating a histogram of them, or showing the most common diagnoses within the window). This would help communicate why it's helpful to annotate a dataset in this way.
(You don't need to use a sample from SynPUF if you don't want to; you could also just construct the simulated data to have more joined diagnoses).

**Comment by eribul** [comment]
Kolla: https://academic.oup.com/jamiaopen/article/2/1/10/5273643

**Comment by eribul** [comment]
Tycks svårt tt få tag på den datan. kan använda exempeldata från `icd.data` ist:
```r
icd.data::uranium_pathology %>% 
  mutate(case = as.character(case)) %>% 
  coder::classify("charlson", id = "case", code = "icd10") %>% 
  coder::index() %>% 
  hist()
```

**Comment by eribul** [comment]
**Review:** Thank you for really good suggestions. Unfortunately, I was not able to include the SynPUF data but I did find a previsously exported data set `icd.data::uranium_pathology` which I have now modified as part of the internal example data sets `ex_icd10` and `ex_people`. I think those data sets should be more realistic. The old vignette was removed as suggested by another review-comment. It is now completelly rewritten and includes some histograms etc as suggested (although with medical ATC-data classified by Risk Rx V).
